Tests/scripts/update_conf_json.py

```diff
@@ -118,18 +118,21 @@ def update_new_conf_json(future):
         pack_integrations, pack_test_playbooks, pack_name = future.result()  # blocks until results ready
         if pack_test_playbooks:
             NEW_CONF_JSON_OBJECT.extend(calc_conf_json_object(pack_integrations, pack_test_playbooks))
+
     except Exception:
         logging.exception('Failed to collect pack test configurations')
-
+        raise
 
 def main():
     install_logging('Update_Tests_step.log', include_process_name=True)
     existing_test_playbooks = load_test_data_from_conf_json()
     with ProcessPool(max_workers=os.cpu_count(), max_tasks=100) as pool:
```

Comment:
Now that we know about this issue in circle - https://github.com/demisto/etc/issues/33244

Lets reduce the number of cpu's to something between 2 to 4.